support/docs/configuration.md

```diff
@@ -1,33 +1,48 @@
-Configuration
+Full configuration
 -------------
+Need to know how to use the transport? [take a look here](transport.md).
```

Comment:
`take a look here` -> `take a look at the transport documentation`

Daarnaast zou ik dit onderaan de pagina zetten. Heeft normaal gesproken niemand nodig.
 En ervan maken:
>This bundle contains a plain smtp transport factory for the Symfony Mailer component, take a look at the transport documentation for more information.